src/containers/PageBuilder/Primitives/SearchCTA/FilterKeyword.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { FieldTextInput } from '../../../../components';
+
+const FilterKeyword = props => (
+  <div className={props.className}>
```

Comment:
Add rootClassName setup.
https://github.com/sharetribe/web-template/blob/710ed304dc2585d49886dd7486fd0b6e79a5201e/src/components/IconSearch/IconSearch.js#L16-L17

Reply:
FYI Filter keyword is in a different branch